anta/cli/nrfu/commands.py

```diff
@@ -105,3 +105,26 @@ def tpl_report(ctx: click.Context, template: pathlib.Path, output: pathlib.Path
     run_tests(ctx)
     print_jinja(results=ctx.obj["result_manager"], template=template, output=output)
     exit_with_code(ctx)
+
+
+@click.command()
+@click.pass_context
+@click.option(
+    "--md-output",
+    type=click.Path(file_okay=True, dir_okay=False, exists=False, writable=True, path_type=pathlib.Path),
+    show_envvar=True,
+    required=True,
+    help="Path to save the report as a Markdown file",
+)
+@click.option(
+    "--only-failed-tests",
+    is_flag=True,
+    default=False,
+    show_envvar=True,
+    help="Only include failed tests in the report.",
+)
```

Comment:
do we need this new option or could we not use the nrfu option

```
  --hide [success|failure|error|skipped]
                                  Hide results by type: success / failure /
                                  error / skipped'.
```

instead and pass it down the line?

Reply:
Agreed. Removed in 8918cd0822dee86db2c1574d30732c0418bfefff